lib/Selection/SelectList.js

```diff
@@ -253,7 +253,6 @@ class SelectList extends React.Component {
                   {ariaLabel => (
                     <input
                       type="text"
-                      role="listbox"
                       aria-label={ariaLabel}
```

Comment:
Thank you @AndreiShumski2 !
NVDA acts the same either way...
JAWS, is very different - it doesn't consider the list.

With this in mind, could you try 
```suggestion
                      aria-label={ariaLabel}
                      role="combobox"
                      aria-controls={`sl-container-${this.props.id}`}
                      aria-expanded="true"
```
axe *may still complain about the `<ul>` needing a label...  it can have the `aria-label`